providers-summary-docs/core-extensions/asset-schemes.rst

```diff
@@ -24,7 +24,7 @@ exposed via Airflow core and community-managed providers.
 ..
   TODO: Change this from Dataset to Asset in Airflow 3.0
 
-See :doc:`documentation on Datasets <apache-airflow:authoring-and-scheduling/datasets>` for details on how dataset URIs work.
+See :doc:`documentation on Assets <apache-airflow:authoring-and-scheduling/asset-scheduling>` for details on how asset URIs work.
```

Comment:
But https://github.com/apache/airflow/blob/097ed5e165773dbf949f4fb7d3eeb63ca055f112/airflow-core/docs/authoring-and-scheduling/datasets.rst is still there

Reply:
Its not present on main: https://github.com/apache/airflow/blob/main/airflow-core/docs/authoring-and-scheduling/datasets.rst doesnt exist but https://github.com/apache/airflow/blob/main/airflow-core/docs/authoring-and-scheduling/assets.rst exists and so does: http://github.com/apache/airflow/blob/main/airflow-core/docs/authoring-and-scheduling/asset-scheduling.rst

Reply:
@amoghrajesh In that case could you add it to https://github.com/apache/airflow/blob/main/airflow-core/docs/redirects.txt

Reply:
@kaxil its already there i think: https://github.com/apache/airflow/commit/1c6509eadd2f30d145067bbc15a3a4a7ba416c38#diff-297ac4bd3f5ca3cfdbacc66e5dbfe086ae3a3b354a5349e4baae4fcfc0133b1eR140

Reply:
Its wrong though -- I think it should link to `airflow-core/docs/authoring-and-scheduling/assets.rst` instead of `airflow-core/docs/authoring-and-scheduling/asset-scheduling.rst`